ChangeLog

```diff
@@ -1,3 +1,28 @@
+2021-09-25  Björn Esser  <besser82 at fedoraproject.org>
+
+	libnss_tcb: Apply proper soname during linking.
+	* libs/Makefile: Apply proper soname to libnss_tcb.so.2.
+
+	libnss_tcb: Drop unneeded LIBNSL from linked libraries.
+	* libs/Makefile: Stop linking libnss_tcb.so.2 against LIBNSL.
+
+	pam_tcb: Allow setting a custom PAM_SO_SUFFIX during build.
+	* pam_tcb/Makefile: Honor PAM_SO_SUFFIX variable.
```

Comment:
IF we really need `PAM_SO_SUFFIX`  support, I suggest to add it in a separate PR.
Background: I introduced `PAM_SO_SUFFIX` in ALT's tcb package long time ago when I was experimenting with OpenPAM, the corresponding changelog entry says:
```
* Thu Jun 05 2003 Dmitry V. Levin <ldv@altlinux.org> 0.9.8.5-alt2
- Fixed build with OpenPAM.
- Added Linux-PAM/OpenPAM multi-build support.
```
Those experiments are mostly forgotten now, and `PAM_SO_SUFFIX` is not very likely to be missed nowadays.

Reply:
Okay, I'll drop it then.  I thought it still is needed for some reason, as the recent package in ALT Linux still carries the patch.